test/integration/targets/win_audit_rule/tasks/main.yml

```diff
@@ -0,0 +1,399 @@
+# - name: register os version (seems integration tests don't gather this fact)
+#   raw: powershell.exe "gwmi Win32_OperatingSystem | select -expand version"
+#   register: os_version
+#   changed_when: False
+# # ^^ seems "raw" is the only module that works on 2008 non-r2. win_command and win_shell both failed
+
+# - name: check if module fails gracefully when older than 2008r2
+#   win_power_plan:
+#   when: os_version.stdout_lines[0] | version_compare('6.1','lt')
+#   check_mode: yes
+#   register: old_os_check
+#   failed_when: old_os_check.msg != 'The win_power_plan Ansible module is only available on Server 2008r2 (6.1) and newer'
+
+- set_fact:
```

Comment:
I would move these to a default file and prefix with `test_audit_rights` so they stay separate from other tests. Also things like the rights and identities set probably just best off people set in the task itself instead of a permission. Makes it easier to see what is actually happening.

Reply:
I kinda wanted to keep them as variables so that it would be easier to change the testing parameters. I think it's still pretty easy to reference by just checking the defaults file. If you really want it changed I can. I implemented the other suggestions though.